elements/nuxeo-browser/nuxeo-breadcrumb.js

```diff
@@ -14,192 +14,247 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+            margin-top: -3px;
+            padding: 0;
+            font-size: 0.75rem;
+          }
 
-      .ancestors a,
-      .breadcrumb-divider {
-        opacity: 0.5;
-        font-weight: 300;
-      }
+          #ancestors li {
+            max-width: 100%;
+            display: inline-block;
+            overflow: hidden;
+            text-overflow: ellipsis;
+            white-space: nowrap;
+          }
 
-      .breadcrumb-divider {
-        vertical-align: middle;
-      }
+          #ancestors li + li::before {
+            content: ' > ';
+            white-space: pre;
+            opacity: 0.5;
+            font-weight: 300;
+          }
 
-      .ancestors a {
-        @apply --nuxeo-link;
-      }
+          #ancestors a,
+          #ellipsis {
+            @apply --nuxeo-link;
+            opacity: 0.5;
+            font-weight: 400;
+            letter-spacing: 0.02rem;
+            text-decoration: none;
+          }
 
-      a:hover {
-        color: var(--nuxeo-link-hover-color);
-        opacity: 1;
-      }
+          #ancestors a:hover {
+            color: var(--nuxeo-link-hover-color, #0066ff);
+            opacity: 1;
+          }
 
-      .doc-path,
-      .ancestors {
-        max-width: 100%;
-        overflow: hidden;
-        text-overflow: ellipsis;
-        white-space: nowrap;
-      }
+          @media (max-width: 1024px) {
+            .current-icon {
+              display: none;
+            }
+          }
+        </style>
 
-      .ancestors span.breadcrumb-item:last-of-type .breadcrumb-divider {
-        display: none;
-      }
+        <nuxeo-connection id="nxcon" url="{{url}}"></nuxeo-connection>
 
-      a.breadcrumb-item-ancestor:nth-last-child(-n + 1),
-      a.breadcrumb-item-ancestor:nth-last-child(-n + 2) a.breadcrumb-item-ancestor:nth-last-child(-n + 3) {
-        display: inline-block;
-      }
+        <div class="breadcrumb">
+          <div class="current-icon">
+            <iron-icon src="[[_icon(document, url)]]"></iron-icon>
+          </div>
+          <div class="doc-path">
+            <a
+              href$="[[urlFor('browse', document.path)]]"
+              class="current breadcrumb-item breadcrumb-item-current"
+              aria-current="page"
+              title="[[_title(document)]]"
+            >
+              [[_title(document)]]
+            </a>
+            <nav aria-label="Breadcrumb">
+              <ol id="ancestors"></ol>
+            </nav>
+          </div>
+        </div>
+      `;
+    }
 
-      .left-ellipsis {
-        direction: rtl;
-        text-align: left;
-      }
+    static get is() {
+      return 'nuxeo-breadcrumb';
+    }
 
-      .right-ellipsis {
-        direction: ltr;
-        text-align: right;
-      }
+    static get properties() {
+      return {
+        document: {
+          type: Object,
+          observer: '_setBreadcrumbElements',
+        },
+        _breadcrumb: {
+          type: Object,
+          value: {},
```

Comment:
This is an array, right?
```suggestion
          type: Array,
```